ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/monitor-layer1.md

```diff
@@ -0,0 +1,54 @@
+# monitor-layer1
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Monitor Layer1 Logging](#monitor-layer1-logging)
+  - [Monitor Layer1 Device Configuration](#monitor-layer1-device-configuration)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## Monitor Layer1 Logging
+
+| Logging |
+| ------- |
+| mac fault |
+| transceiver dom |
+| transceiver communication |
```

Comment:
Maybe it should come with a column thats says "Enabled" and "true/false" or something like "state" and "enabled/disabled" ?